README.md

```diff
@@ -4,4 +4,4 @@ This should build a custom Linux image that can be pushed to Azure Community Gal
 
 # Release Instructions
 
-CI is set to run when a tag is created in trunk. Increment the version number of the previous tag to release. Use SemVer (vX.X.X). The CI run will upload the image with version X.X.X to match their image versioning requirements, Major(int).Minor(int).Patch(int) .
\ No newline at end of file
+CI is set to run when a tag is created in trunk. Increment the version number of the previous tag to release. Use SemVer (vX.X.X). The CI run will upload the image with version X.X.X to match their image versioning requirements, Major(int).Minor(int).Patch(int).
```

Comment:
https://learn.microsoft.com/en-us/azure/container-registry/container-registry-image-tag-version

Azure gallery uses Major.Minor versioning not semver. The tagging should mirror this as it the deploy will fail otherwise.